tracer/src/Datadog.Tracer.Native/debugger_tokens.cpp

```diff
@@ -69,6 +72,10 @@ HRESULT DebuggerTokens::WriteLogArgOrLocal(void* rewriterWrapperPtr, const TypeS
 
         // DebuggerState
         signature[offset++] = ELEMENT_TYPE_BYREF;
+        if (isMultiProbe)
```

Comment:
How come this array is created in addition to, and not instead of, the regular `MethodDebuggerState` parameter?

Reply:
In .NET instrumentation if you want an array of certain type you have to prefix the type with `ELEMENT_TYPE_SZARRAY`.

Reply:
Got it, thanks for clarifying!